agent/utils/dockerapi_compare_versions.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2014-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Would it be better to place this in our docker client package rather than utils?

Reply:
placed the files in /agent/dockerclient path